cli/pack/app_dir.go

```diff
@@ -336,6 +336,10 @@ func copyProjectFiles(dst string, ctx *context.Ctx) error {
 				return false, fmt.Errorf("Failed to get file rel path: %s", err)
 			}
 
+			if strings.HasPrefix(relPath, "tmp/") {
```

Comment:
Please, comment in the code on what you are doing and why. It will be useful in the future.

Reply:
``run_dir`` (by default) is in ``tmp`` folder. But, besides ``run_dir``, ``data and ``log`` are also stored there. I'm not sure if we can ignore them either.

Reply:
Cartridge CLI allows to describe ``run_dir`` in two ways:

* pass ``--run_dir`` flag directly
* describe it in ``.cartridge.yml`` file

In second we case we can get ``run_dir`` from .cartridge.yml (and ignore this folder). 

But, in first case, if you user always specify this parameters with ``--run_dir`` flag, we cant determine the real ``run_dir``. 

I suggest to just ignore the ``/tmp`` folder (like in current patch). Or ignore ``/tmp`` and parameters set in ``.cartridge.yml``. I think first option is better.

Reply:
I added ignoring ``tmp/data``, ``tmp/log``, ``tmp/run`` to the code.

> We can't understand inside the program with what flag it was run? Sounds strange.

Yes, because this flags specifying when user type ``cartridge start`` command (not ``cartridge pack``.